examples/cassandra/README.md

```diff
@@ -57,103 +71,51 @@ command line tool somewhere in your path.  Please see the
 for installation instructions for your platform.
```

Comment:
Yes, this is probably a general issue when people visit any of these examples.  OTOH we don't want to repro those cluster startup instructions for every example, especially since different users will be wanting to use different cloud providers (AWS etc.). Perhaps we should do more to create a kind of "guided path" through the examples, starting with the "setup" ones. We do this to an extent.